app/propsGenerators/indexPagePropsGenerator.ts

```diff
@@ -19,16 +20,20 @@ function calculateDosePercentage(
   return parseFloat(((dosesCount / totalPopulation) * 100).toFixed(6));
 }
 
+const getLatestFetchyRunCache = createCache(getLatestFetchyRun, {
+  name: "latest-fetchy-run",
+  timeToLive: 5 * 60,
```

Comment:
Minor/preference: shall we just use the abbreviation here? Feel like `ttl` is fairly common?